documentation/book/cluster-operator.adoc

```diff
@@ -78,6 +78,26 @@ A JSON string representing the storage configuration for the Kafka broker nodes.
 `kafka-metrics-config`::
 A JSON string representing the JMX exporter configuration for exposing metrics from Kafka broker nodes.
 When this field is absent no metrics will be exposed.
+`kafka-logging`::
+A JSON string representing the level of logging for Kafka nodes. This field can contain JSON with loggers, which should be changed and levels.
+Or there can be specified name of config map, which should be used for setting logger levels.
+When this field is absent default values are used.
+List of loggers and field, which can be set:
+`
+kafka.root.logger.level
+log4j.logger.org.I0Itec.zkclient.ZkClient
+log4j.logger.org.apache.zookeeper
+log4j.logger.kafka
+log4j.logger.org.apache.kafka
+log4j.logger.kafka.request.logger
+log4j.logger.kafka.network.Processor
+log4j.logger.kafka.server.KafkaApis
+log4j.logger.kafka.network.RequestChannel$
+log4j.logger.kafka.controller
+log4j.logger.kafka.log.LogCleaner
+log4j.logger.state.change.logger
+log4j.logger.kafka.authorizer.logger
+`
```

Comment:
Although each logger is given its own line here, they are concatenated together when rendered in the docs. I think it has to do with the ` characters. Is this the intended behaviour?